neurodamus/metype.py

```diff
@@ -315,24 +298,6 @@ def _make_coord_map_matrix(position, rotation, scale):
         m[:, 3] *= scale
         return m
 
-    def local_to_global_coord_mapping(self, points):
```

Comment:
this is used in multiscale run (if only ngv was working):

https://github.com/openbraininstitute/MultiscaleRun/blob/45c493da4e6c9ec604c672864b6927a3b5e34977/multiscale_run/neurodamus_manager.py#L191

Reply:
Ok, I've put them back, and created a ticket: https://github.com/openbraininstitute/neurodamus/issues/320